jenkins/controller/plugins.txt

```diff
@@ -9,7 +9,7 @@ configuration-as-code-groovy:1.1
 timestamper:1.11.8
 build-token-root:1.7
 github:1.33.1
-workflow-api:2.46
```

Comment:
Releases changed recently to be GitHub Actions triggered: https://github.com/jenkinsci/workflow-api-plugin/releases. See also https://github.com/jenkinsci/workflow-api-plugin/pull/183.